src/concept/ConceptManager.jl

```diff
@@ -19,7 +19,7 @@ function Base.get(cm::ConceptManager, ::Type{EntityType}, label::String)
     return nothing
 end
 
-function put(cm::ConceptManager, ::Type{EntityType}, label::String)
+function put(cm::ConceptManager, ::Type{EntityType}, label::Optional{String})
```

Comment:
How do we handle `nothing`?

Reply:
Same as in the put_entity_req. Probably this is still a remnant of problem solving trial.